tests/bismark/main.nf.test

```diff
@@ -24,19 +24,22 @@ nextflow_pipeline {
                 new File(".nf-test/indexes/BismarkIndex/").getAbsolutePath()
             )
 
-            // Test results
+            // Assert Workflow Success
             assert workflow.success
-            assert snapshot(workflow.trace.tasks().size()).match()
-            // FIXME assert snapshot(workflow).match()
-            assert new File("$outputDir/multiqc/bismark/multiqc_report.html").exists()
 
-            assert new File("$outputDir/bismark/methylation_calls/splitting_report/SRR389222_sub1_trimmed_bismark_bt2.deduplicated_splitting_report.txt").exists()
-            assert new File("$outputDir/bismark/methylation_calls/splitting_report/SRR389222_sub2_trimmed_bismark_bt2.deduplicated_splitting_report.txt").exists()
-            assert path("$outputDir/bismark/methylation_calls/splitting_report/SRR389222_sub2_trimmed_bismark_bt2.deduplicated_splitting_report.txt").md5 == "3038a90a6e513c3b1c0de540439eb101"
+            // Snapshot
+            assert snapshot(path("$outputDir/bismark/methylation_calls/splitting_report/Ecoli_10K_methylated_1_val_1_bismark_bt2_pe.deduplicated_splitting_report.txt"),
+                            path("$outputDir/bismark/methylation_calls/splitting_report/SRR389222_sub1_trimmed_bismark_bt2.deduplicated_splitting_report.txt"),
+                            path("$outputDir/bismark/methylation_calls/splitting_report/SRR389222_sub2_trimmed_bismark_bt2.deduplicated_splitting_report.txt"),
```

Comment:
Are these reproducible? I don't think I could get a consistant md5sum out of them. Or is there some way to turn the md5sum off in the snapshot?

Reply:
i am getting consistent md5s for these files. Could you give it a test @Emiller88 ?

Reply:
If you and the CI are passing then I think I'd be the odd one out anyway, I think it's good then!